versions.yaml

```diff
@@ -219,6 +219,13 @@ externals:
     version: "42b7c9687ecd0907ef70da31cf290a60ee8432cd"
     toolchain: "1.72.0"
 
+  coco-kbs:
+    description: "Provides attestation and secret Management services"
+    url: "https://github.com/confidential-containers/kbs"
+    version: "112018371d38c73eba8998f4f6b81ebe00b41556"
+    image: "ghcr.io/confidential-containers/staged-images/kbs"
+    image_tag: "112018371d38c73eba8998f4f6b81ebe00b41556"
```

Comment:
Let's remember to bump this after we merge your KBS PR. 

At some point we should use the KBS releases, but this good for now.

Reply:
Bumped on https://github.com/kata-containers/kata-containers/pull/9065 . Thanks for the reminder!